test/jobs/channels/transfer_channels_job_test.rb

```diff
@@ -28,11 +28,21 @@ class TransferChannelsJobTest < ActiveJob::TestCase
     contested_by_channel = channels(:locked_out_site)
     # contest channel
     Channels::ContestChannel.new(channel: channel, contested_by: contested_by_channel).perform
+    clear_enqueued_jobs
+    clear_performed_jobs
 
-    travel (Channel::CONTEST_TIMEOUT + 1.minute) do
-      assert_enqueued_jobs(6) do
-        TransferChannelsJob.perform_now
-      end
+    travel(Channel::CONTEST_TIMEOUT + 1.minute) do
+      assert_enqueued_jobs 0
+      Channels::TransferChannelsJob.perform_now
+
+      assert_enqueued_jobs 1
+      assert_performed_jobs 0
+
+      perform_enqueued_jobs
+
+      # There are 5 jobs enqueued (4 emails and a slack message) when a channel completes transfer
+      # + 2 for the actual jobs that were completed, so 7 overall
+      assert_performed_jobs 7
```

Comment:
It looks like we removed the slack messenger call so wouldn't the jobs be lower?

Reply:
Normally yes, but we converted `Channels::ApproveChannelTransferJob` to be a job and called `perform_later` to enqueue it 😄

Reply:
@corymcdonald can you adjust the comment above to match?

Reply:
We actually still do send a slack message during this process because we have an `after_save` event on `Channel` which triggers when the transferred channel verifies 😩 

https://github.com/brave-intl/publishers/blob/staging/app/models/channel.rb#L58